internal/server/server.go

```diff
@@ -172,3 +174,31 @@ func (s ServerForClient) GetGameState() gamestate.ClientGameState {
 func (s ServerForClient) GetGameConfig() config.ClientConfig {
 	return s.server.gameConfig.GetClientConfig()
 }
+
+func randomAssign(input []shared.ClientID) (shared.ClientID, shared.ClientID, shared.ClientID) {
+	lst := arrayCopy(input)
+
```

Comment:
```suggestion
	lst = make([]shared.ClientID, len(input))
	copy (lst, input)
```

Reply:
Perfect, thanks! Been trying to find something better